File: abc/abc294/a.cpp
// https://atcoder.jp/contests/abc294/submissions/39841434

#include <bits/stdc++.h>
using namespace std;

int main() {
  int n;
  cin >> n;

  vector<int> a(n);
  for(auto &z : a) {
    cin >> z;
    if(z % 2 == 0) cout << z << " ";
  }
}

File: abc/abc294/a.cs
// https://atcoder.jp/contests/abc294/submissions/46505564

using System;

public class A
{
  static void Main()
  {
    int n = int.Parse(Console.ReadLine());
    string[] a = Console.ReadLine().Split(" ");

    for(int i = 0; i < n; i++)
    {
      if(int.Parse(a[i]) % 2 == 0)
      {
        Console.Write(a[i] + " ");
      }
    }
  }
}

